os/options.go

```diff
@@ -44,6 +45,7 @@ func (parser *EnvOptionParser) Options() (*agent.Options, error) {
 		EdgeInactivityTimeout: agent.DefaultEdgeSleepInterval,
 		EdgeInsecurePoll:      false,
 		LogLevel:              agent.DefaultLogLevel,
+		DockerBinaryPath:      os.Getenv(EnvKeyDockerBinaryPath),
```

Comment:
No need to provide the Docker binary path as an option. It will be hardcoded to `/app/docker` or (`/app/docker.exe` on windows)

Reply:
sounds good. Where should I hardcode it? into the stack manager?

Reply:
Should be hardcoded in the domain (`agent.go` with the other constants).